findbugs.properties

```diff
@@ -1,11 +1,17 @@
 category=External Analysers
 description=Provide Findbugs rules for analysis of Java projects
-archivedVersions=2.4,3.0,3.1,3.2,3.3,3.4.3,3.4.4,3.5,3.6,3.7,3.8.0,3.9.0,3.9.1,3.9.2,3.9.3,3.10.0
-publicVersions=3.9.4,3.11.0
+archivedVersions=2.4,3.0,3.1,3.2,3.3,3.4.3,3.4.4,3.5,3.6,3.7,3.8.0,3.9.0,3.9.1,3.9.2,3.9.3,3.10.0,3.11.0
+publicVersions=3.9.4,3.11.1
 
 defaults.mavenGroupId=org.sonarsource.sonar-findbugs-plugin
 defaults.mavenArtifactId=sonar-findbugs-plugin
 
+3.11.1.description=Use SpotBugs 3.1.12
+3.11.1.sqVersions=[7.6,LATEST]
+3.11.1.date=2019-09-23
+3.11.1.changelogUrl=https://github.com/spotbugs/sonar-findbugs/releases/tag/3.11.1
+3.11.1.downloadUrl=https://github.com/spotbugs/sonar-findbugs/releases/download/3.11.1/sonar-findbugs-plugin-3.11.1.jar
+
 3.11.0.description=Use SpotBugs 3.1.12
 3.11.0.sqVersions=[7.6,LATEST]
```

Comment:
Only one version can be compatible with `LATEST`. The convention is to edit the old version's range end to the SQ version current at the time.